lib/Test/Nginx/Util.pm

```diff
@@ -604,10 +604,15 @@ sub kill_process ($$$) {
                 warn "waitpid timeout: ", timeout();
             }
 
-            local $SIG{ALRM} = sub { die "alarm\n" };
```

Comment:
If we remove the calling of `die`, `eval {}` block and `if ($@) {}` is useless, just remove it